RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGeneratorGPU.cc

```diff
@@ -93,15 +103,18 @@ void CAHitQuadrupletGeneratorGPU::fillDescriptions(edm::ParameterSetDescription
   desc.add<double>("hardCurvCut", 1.f / (0.35 * 87.f))->setComment("Cut on minimum curvature");
   desc.add<double>("dcaCutInnerTriplet", 0.15f)->setComment("Cut on origin radius when the inner hit is on BPix1");
   desc.add<double>("dcaCutOuterTriplet", 0.25f)->setComment("Cut on origin radius when the outer hit is on BPix1");
-  desc.add<bool>("earlyFishbone", false);
-  desc.add<bool>("lateFishbone", true);
+  desc.add<bool>("earlyFishbone", true);
+  desc.add<bool>("lateFishbone", false);
   desc.add<bool>("idealConditions", true);
   desc.add<bool>("fillStatistics", false);
   desc.add<unsigned int>("minHitsPerNtuplet", 4);
+  desc.add<bool>("noForwardTriplets", true);
```

Comment:
I find this parameter slightly confusing. I find it typically more clear to avoid the negation in the parameter name (i.e. e.g. `doForwardTriplets`), but then I'm further confused what you wrote in the description "set to False to add jumping doublets in the forward (just FPIX1,FPIX3)". Should it then be called e.g. `includeJumpingDoublets`?

Reply:
is indeed includeJumpingForwardDoublets ...
jumping Barrel Doublets are enabled by default fi triplets are requested...
will change